src/networking.c

```diff
@@ -2758,18 +2758,18 @@ NULL
     }
 }
 
-/* HELLO <protocol-version> [AUTH <user> <password>] [SETNAME <name>] */
+/* HELLO [protocol-version] [AUTH <user> <password>] [SETNAME <name>] */
```

Comment:
other arguments are only possible if protocol version is given.
```suggestion
/* HELLO [<protocol-version> [AUTH <user> <password>] [SETNAME <name>] ] */
```